### PR TITLE
Feat/account switch refactor

### LINKED ISSUE #555: Select Account UI Update
# Description

We want to update the way the extension handles the account switching between subaccount in the wallet

[https://www.figma.com/file/no61f8MMPQlPUIaXSX3zCn/Plug---Wallet?node-id=8117%3A1877](https://www.figma.com/file/no61f8MMPQlPUIaXSX3zCn/Plug---Wallet?node-id=8117%3A1877)

### PR DESCRIPTION
**Pull request by angarita-dev**
## Changelog

- Changed `Profile view`
- Changed `Wallet details`

### Type of changes included:

- [x] Components
- [x] Business Logic
- [ ] Bug fixes
- [ ] Styling
- [x] Code Refactor

### Clubhouse Tickets Related:

- #555

### Screenshots/Gifs:

image
image

### Notes:

*Place special notes here*

### Tested on:

- [x] Chrome
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: source/components/Profile/components/AccountItem/styles.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    color: 'rgba(0, 0, 0, 0.4)',
+  },
+  rightContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+    opacity: '0',
+
+    transition: 'all 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms',
```
Comment (tomiir) [inline source/components/Profile/components/AccountItem/styles.js]: would be cool to abstract this to a generic animation , do you know if it's being used someplace else?

File: source/components/Profile/components/AccountItem/styles.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    '& > button': {
+      padding: '5px',
+      '& > span': {
+        width: '24px',
+        height: '24px',
+      },
+    },
```
Comment (tomiir) [inline source/components/Profile/components/AccountItem/styles.js:65]: are these needed? don't they have a style on their own?

File: source/components/Profile/index.jsx [hunk 3]
```diff
@@ -272,31 +263,19 @@ const Profile = ({ disableProfile }) => {
...
-                        ) : null}
+                    const isCurrentAccount = account.walletNumber === walletNumber;
+
+                    return (
+                      <AccountItem
+                        account={account}
+                        isEditing={isEditing}
+                        isCurrentAccount={isCurrentAccount}
+                        handleChangeAccount={handleChangeAccount}
+                        handleEditAccount={handleEditAccount}
```
Comment (tomiir) [inline source/components/Profile/index.jsx:274]: thank u

**Review by 0xAaCE — APPROVED** [review]
LGTM

**Review by tomiir — APPROVED** [review]

File: source/components/Profile/components/AccountItem/styles.js (hunk 1 above)
Comment (angarita-dev) [reply source/components/Profile/components/AccountItem/styles.js]: Done 🚀

File: source/components/Profile/components/AccountItem/styles.js (hunk 2 above)
Comment (angarita-dev) [reply source/components/Profile/components/AccountItem/styles.js:65]: Added it that way to add styles to a mui component without having to hassle with two extra classes

**Pull request merged**